### ISSUE TITLE
Inline Type Definitions also in chat

### ISSUE THREAD

**Issue by denisdasilvarocha** [cursor/cursor#1467]
**Describe the solution you'd like**
It would be really great to have Type Definitions in chat aswell in the way we have as inline feature, because this way we can quickly have the entire context and prompt right at our fingertips.
![Screenshot 2024-05-25 at 14 23 33](https://github.com/getcursor/cursor/assets/92863811/18619eab-258b-4507-88cf-b52682dc6874)

**Comment by andrew-healey** [comment]
I like this. Specifically, would you want definitions for the code selections you've attached to the chat?

**Comment by Dreamzlol** [comment]
> I like this. Specifically, would you want definitions for the code selections you've attached to the chat?

Yeah, that would be absolutely awesome, that way he has context about everything related to the code.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
